test/modules/core/shadertools/fp64/fp64-test-utils-transform.js

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2015 - 2017 Uber Technologies, Inc.
```

Comment:
Nit: Is all this code new? Looks like a modified version of existing code. Forgot to delete the old versions?

Reply:
Left the old ones as is, since `Transform` based can only be run using WebGL2 only. Once we add WebGL1 fallback, these old test files can be removed, I left a TODO. Addressed all other comments.